apps/fluid-dev/src/app/app.component.html

```diff
@@ -14,6 +14,7 @@
     <li><a routerLink="/button">Button</a></li>
     <li><a routerLink="/button-group">Button-Group</a></li>
     <li><a routerLink="/checkbox">Checkbox</a></li>
+    <li><a routerLink="/combobox">ComboBox</a></li>
```

Comment:
Maybe consider spelling it Combobox with a lower case b instead of ComboBox (and `fluid-combobox` instead of `fluid-combo-box`)? It would be consistent with Checkbox and I think I've mostly seen the latter variant so I would probably introduce typos with your spelling, though your mileage may vary. You've also named this route `combobox` but the folder `combo-box`.

Reply:
good point, I don't really have an opinion on that, but I would definitely agree to keep things consistent, yeah. will rename :)